Hardware/Hardware/CardReader/CardReaderEvent.swift

```diff
@@ -15,6 +15,9 @@ public enum CardReaderEvent {
     /// A card was removed.
     case cardRemoved
 
+    /// A card was removed after client-side payment capture.
+    case cardRemovedAfterPaymentCapture
```

Comment:
I have a question here :) Does payment capture imply client-side payment capture semantics as the comment explains? Otherwise, shall we add it to the case naming? `cardRemovedAfterClientSidePaymentCapture`

Reply:
Makes sense! Renamed in https://github.com/woocommerce/woocommerce-ios/pull/6738/commits/e1678504965ebf5a3568575e4d31d177ca02db7a